src/models/ProductResponse.ts

```diff
@@ -0,0 +1,7 @@
+export interface ProductResponse {
+  Id:number;
+  name:string;
+  price:number;
+  discountPrice?:number | null;
```

Comment:
by the way, do you really need null?

Reply:
When I wrote the products, I wanted to write a condition so that the does not come when discount is null, but the condition did not work, just yesterday I wrote this to you in telegram